package.json

```diff
@@ -76,6 +76,7 @@
         "stylelint": "^15.2.0",
         "stylelint-config-standard-scss": "^7.0.1",
         "swc-loader": "^0.2.3",
+        "tiny-invariant": "^1.3.1",
```

Comment:
This is a [very small](https://github.com/alexreardon/tiny-invariant/blob/master/src/tiny-invariant.ts) package that is [pretty widely used](https://www.npmjs.com/package/tiny-invariant)

- make assertions concisely
- typescript narrowing

It is similar to `assertNotNil` and `assertInstanceOf`, though I probably would not have written those functions if I was familiar with `tiny-invariant` at the time.

```
assertNotNil(x) <---> invariant(x)
assertInstanceOf(x, SomeClass) <---> invariant(x instanceof SomeClass)
```